src/main/java/core/basesyntax/strategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy;
+
+import java.util.List;
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy {
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public void accept(List<String[]> data, Map<String, OperationHandler> operationHandlerMap) {
```

Comment:
map should be class-level field initialized through the constructor and this method should just return the corresponding handler and nothing else
Handlers `accept()` method may be called in another service like `TransactionProcessingService` where you will iterate over the transactions list
```suggestion
    public OpearationHandler getHandlerByOperation(FruitTransaction.Operation operation) {
```